app/domain/people_search.rb

```diff
@@ -35,19 +40,19 @@ def pre_load(people)
     person_keys = people.map(&:id)
 
     Person.includes(:department, :roles, :projects, :activities,
-                    :educations, :advanced_trainings, :expertise_topics)
+                    :educations, :advanced_trainings, (:skills if search_skills))
           .find(person_keys)
   end
 
   def in_associations(person)
-    association_symbols.each do |sym|
-      attribute_name = in_association(person, sym)
-      if attribute_name
-        return format('%<association>s#%<attribute_name>s',
-                      association: sym.to_s, attribute_name: attribute_name)
+    association_symbols.map do |sym|
+      attribute_names = in_association(person, sym)
+      attribute_names.map do |attribute_name|
+        format('%<association>s#%<attribute_name>s',
```

Comment:
why not just `"#{sym.to_s}\##{attribute_name}"` 
Doesn't this do the same or am I missing something?